pkg/devicemapper/devmapper.go

```diff
@@ -264,14 +259,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-var dmLogger DevmapperLogger
-
-// LogInit initializes the logger for the device mapper library.
-func LogInit(logger DevmapperLogger) {
-	dmLogger = logger
-	LogWithErrnoInit()
-}
```

Comment:
Just to be clear, I do understand wanted to provide logging hooks, so maybe we should re-add it. I'm just trying to point out that the old API was not correct. Sorry for the strong reply.